src/07-even-more-loops.js

```diff
@@ -57,7 +76,10 @@ function shoutForOfLoop(array) {}
  *  sumArray([ 10, 0, 10, 11 ]);
  *  //> 31
  */
-function sumArray(array) {}
+function sumArray(array) {
+  if (!array.length) return 0;
+  return array.reduce((a, b) => a + b);
```

Comment:
Great use of `reduce` like other higher order functions just be clear that we know what exactly is happening at each step.